### PR TITLE
webgpu: add fusedDepthwiseConv2D operator

### PR DESCRIPTION
**Pull request by xhcao**
DEV

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4468)

### CODE REVIEW

**Comment by xhcao** [comment]
@qjia7 @axinging @haoyunfeix @gyagp Please take a look, thank you.

**Comment by qjia7** [comment]
LGTM. @pyu10055 @lina128 Please take a look. Thanks.

**Review by qjia7 — APPROVED** [review]
+@kainino0x  take another look, thanks.

**Review by lina128 — APPROVED** [review]
LGTM

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4468#-:-MQoRPz-CPS-NIOR-05L:bb6ywmx)* status: :shipit: complete! 2 of 1 approvals obtained

File: tfjs-backend-webgpu/src/kernels/FusedDepthwiseConv2D.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  backend: WebGPUBackend
+}) {
+  const {inputs, backend, attrs} = args;
+  const {x, filter, bias, preluActivationWeights} = inputs;
+  const {strides, pad, dilations, dimRoundingMode, activation} = attrs;
+
+  let $dilations = dilations;
+  if ($dilations == null) {
+    $dilations = [1, 1];
+  }
```
Comment (kainino0x) [inline tfjs-backend-webgpu/src/kernels/FusedDepthwiseConv2D.ts:35]: Could this be simply:
```suggestion
  const {strides, pad, dilations = [1, 1], dimRoundingMode, activation} = attrs;
```
?

**Review by kainino0x — APPROVED** [review]

File: tfjs-backend-webgpu/src/kernels/FusedDepthwiseConv2D.ts (hunk 1 above)
Comment (lina128) [reply tfjs-backend-webgpu/src/kernels/FusedDepthwiseConv2D.ts:35]: If dilations = null, the default won't apply.

Comment (kainino0x) [reply tfjs-backend-webgpu/src/kernels/FusedDepthwiseConv2D.ts:35]: Aha, good point, I missed that.

**Pull request merged**